test/src/main/java/io/strimzi/test/k8s/KubeClient.java

```diff
@@ -93,7 +93,7 @@ default K delete(String... files) {
      * @param command The command
      * @return The process result.
      */
-    ProcessResult execInPod(String pod, String... command);
+    ExecResult execInPod(String pod, String... command);
```

Comment:
Why rename the class?

Reply:
It make more sense to me - we have `Exec` as an executor so results of this should be `ExecResult`